guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,7 +4,11 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== postgresql-evr extension no longer required
+
+Installation of the Katello database on remote systems where root access is not available is now possible.
+Only a basic PostgreSQL installation is required, which should enable installs on systems like Amazon RDS or Azure Database for PostgreSQL.
```

Comment:
This is out of scope, but we have a guide https://docs.theforeman.org/nightly/Deploying_Project_on_AWS/index-katello.html and using Amazon RDS would be a good thing to document, after it's been verified.

Reply:
@maximiliankolb Just FYI, this might be useful to know with regards to your plans for the AWS guide.